src/test/java/io/strimzi/test/container/StrimziKafkaContainerIT.java

```diff
@@ -20,20 +21,17 @@
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.junit.jupiter.api.Assertions.assertThrows;
 
-public class StrimziKafkaContainerIT {
+public class StrimziKafkaContainerIT extends AbstractIT {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(StrimziKafkaContainerIT.class);
 
     private StrimziKafkaContainer systemUnderTest;
 
-    private void assumeDocker() {
-        Assumptions.assumeTrue(System.getenv("DOCKER_CMD") == null || "docker".equals(System.getenv("DOCKER_CMD")));
-    }
-
-    @Test
-    void testStartContainerWithEmptyConfiguration() {
+    @ParameterizedTest(name = "testStartContainerWithEmptyConfiguration-{0}")
+    @MethodSource("retrieveKafkaVersionsFile")
```

Comment:
Can't this be set on the class level?

Reply:
I asked the same question to myself but this `@MethodSource` can be only set in method scope. :( Same applies to `@ArgumentsSource` and other source types, which JUnit5 provides... 

Another approach would be using `@TestFactory` where we could eliminate multiple calls of the retrieveKafkaVersionsFile method...but I don't know if that's worth it...because it would make tests harder to understand :)

Reply:
Well, I guess you can keep this then.